Add GDPR privacy policy v1

GregSutcliffe opened the pull request:
This is the first version of our privacy policy. I've left this rather late (GDPR starts today :P), so unless there's *glaring* omissions I'd like to get this merged and then evolve it. I'll send a summary to the forum once it's live.

It covers everything I can think of, and attempts to do so in plain language rather than legalese. The areas it covers were inspired by the work in the [Fedora policy](https://fedoraproject.org/wiki/Legal:PrivacyPolicy) and the [Cockpit policy](https://cockpit-project.org/privacy.html) which is a simplified version. However, the style and language is inspired by [this policy](https://juro.com/policy.html) which I regard as *far* more readable.

GregSutcliffe committed:
Add GDPR privacy policy v1

tbrisker reviewed (approved):
IANAL but LGTM

ohadlevy commented on privacy.md:
```diff
@@ -0,0 +1,165 @@
+information or set cookies. There is no account for the website itself. The
+webserver (Apache) logs will contain IP addresses, timestamps, and UserAgent
+strings - these are log-rotated and deleted after 4 weeks.
+
```
We have Google analytics enabled for the site.

GregSutcliffe replied:
We don't actually use it at the moment, I can't recall the last time I looked at it. we should disable it. And Google+ integration ;)

ohadlevy replied:
well... I used it last week for someone asking how many IE users we have, so YMMV

GregSutcliffe replied:
sorry, only saw this after I finished my updates, lag--. I'm honestly unsure of the privacy impact here, I saw some of the discussion around google analytics and it made me quite uncomfortable. If you can draft something to include here I'll be happy to include it, but this has to go out today / tomorrow so I can include it in the newsletter.

dhollinger commented on privacy.md:
```diff
@@ -0,0 +1,165 @@
+
+* The right to access your personal data
+* The right to rectify the personal data we hold about you
+* The right to erase your personal data
```
How does the Foreman project plan to handle "Right to erasure" requests that relate to Git commit history?

Will the project rewrite commit history to remove every instance of a user's email and force all other users to force pull/rebase or will the project take a stance similar to [Gitlab](https://about.gitlab.com) and require that a user forfeits their right to erasure on commit history data as it could be seen as detrimental to the project, code base, and overall product to rewrite the commit history?

ekohl replied:
I don't think we should rewrite. There are too many issues with it.

GregSutcliffe replied:
I think you can argue both "consent" and "legitimate interest" grounds for not rewriting it.

The user has full control over what is submitted to the Git repo, including the option to use psudeonymous contact information, so providing your real name / email is opt-in, and you're not being denied the service if you don't want to.

We also would like contact information, particularly in the case (as other projects have had) where you need to relicense, and thus contact all the contributors. Also if we need to show provenance of intellectual property for any reason. So there's legitimate interest of the project too.

We do need to be clear that this is the case though, so I'll update the text. Thanks for the heads up

GregSutcliffe replied:
This is a nice reference, by the way - https://law.stackexchange.com/a/28858

ekohl commented on privacy.md:
```diff
@@ -0,0 +1,165 @@
+
+* Legal basis for this data usage: Legitimate interest
+
+The website does embed YouTube videos, see [YouTube's privacy policy][YouTube]
```
It also embeds from other sources like video.fosdem.org and slideshare.net on the media page.

GregSutcliffe replied:
ty, will go on a link-gathering hunt. we should consider re-doing how we handle videos at some point ;)

GregSutcliffe replied:
Fosdem doesn't seem to have a privacy policy, i doubt they capture anything at all. linked to the homepage for now

ekohl commented on privacy.md:
```diff
@@ -0,0 +1,165 @@
+* Legal basis for this data usage: Legitimate interest
+
+The Foreman community also run a bug tracker using the
+[Redmine](http://www.redmine.org) software. When you sign up for a Redmine
```
https://

ekohl commented on privacy.md:
```diff
@@ -0,0 +1,165 @@
+*public* channels are logged to [BotBot.me](https://botbot.me) who also have a
+[privacy policy][BotBot].
+
+The Foreman community uses a forum using the [Discourse](https://discourse.org)
```
Should it be noted that this is self-hosted? The current text could be interpreted as hosted by discourse.org.

ekohl commented on privacy.md:
```diff
@@ -0,0 +1,165 @@
+The Foreman community also run a bug tracker using the
+[Redmine](http://www.redmine.org) software. When you sign up for a Redmine
+account, you provide us with an email address and a name. We only use this
+email within Redmine to contact you about bugs you have reported or subscribed
```
This email address is also used to link to git authors. It can be linked manual as well. This allows us to let you close an issue from a git commit with the correct attribution. We also build a more complete user activity because of this.

GregSutcliffe replied:
Thanks, I've restructed the Redmine section for this.

ekohl commented:
For downloads.theforeman.org we use fastly as a CDN. That can also log IPs for requests.

ares commented on privacy.md:
```diff
@@ -0,0 +1,165 @@
+
+## Surveys
+
+The Foreman community runs a survey each year. Completing the survey is op-in,
```
s/op-in/opt-n/

ares commented:
IMHO a good version to start with, I think once suggestions above are incorporate, this could be merged.

GregSutcliffe committed:
GDPR review comments round 1

GregSutcliffe commented:
Ok pushed a second commit so that people can easily review the changes. Keep it coming :)

GregSutcliffe commented:
I've added a commit removing all the Google Analytics. We don't currently use it, and it's a privacy nightmare if you ask me. When we're ready, I'll add an FOSS analytics system like Piwik and we can update this poilcy :)

GregSutcliffe commented:
Just so other commenters are aware, @ohadlevy and I are disagreed on the value of Google Analytics, see the comment near the top. Other thoughts would be great :)

GregSutcliffe committed:
Anonymise IP in Google Analytics and remove other tracking

* Remove strut.io tracking in slides
* Remove Twitter blog integration
* Remove defunct Google+ blog integration

dLobatog commented on privacy.md:
```diff
@@ -0,0 +1,210 @@
+
+Using a VPN, Tor, or other IP / UserAgent masking service will not affect your
+use of the project website, as it is entirely static. For Redmine & Discourse,
+we do not mandate use of real names, and using psudonyms will not affect the
```
typo, `pseudonyms`

GregSutcliffe replied:
thanks fixed

dLobatog commented on privacy.md:
```diff
@@ -0,0 +1,210 @@
+### Your accounts on our services
+
+Your Redmine & Discourse accounts contain options to control what emails are
+sent to you. 3rd party services such as [GitHub](https://github.com) retain
```
What is the default? AFAIK it should be *send nothing* unless the user configures it

GregSutcliffe replied:
I believe the default is "things you chose to interact with" - i.e bugs you've written or commented on, posts you've replied to. Discourse definitely has nothing in the email section unless you go add it.

GregSutcliffe replied:
Oh, the Discourse weekly digest is auto-enabled, but I would argue that's OK under legitimate interest of building a community - an infrequent reminder about the forum (which you can, of course, disable) is consistent with that.

(Note, you only see that email at all if you stop interacting with the forum, so it won't spam you if you're a regular. If you've no idea what I'm talking about, well done on consistently interacting with the forum :P)

dLobatog reviewed (commented):
I've just left a question about sending emails inline, outstanding work @GregSutcliffe !

GregSutcliffe committed:
Add Google Analytics paragraph

GregSutcliffe commented:
OK, after an extended discussion with @ohadlevy and a dig into what Google Analytics config we have, I think we're sorted. I've reverted the previous commit, and re-instated GA with `anonymiseIP` enabled.

I've also been through [these slides (pdf)](https://webdevlaw.uk/wp-content/uploads/2018/05/WordCamp-Belfast-GDPR-Google-Analytics.pdf) from one of my personal tech policy heroes on how to do GA right - happily we were already doing most of that.

I've also been reading [this interesting interpretation](https://brianclifton.com/blog/2018/04/16/google-analytics-gdpr-and-consent) which suggests that if GA is *all* we do, then we're all set. Since we don't make much use of the Twitter account, and our G+ is about to be replaced with Discourse for blog comments, I've gone ahead and removed them. That only leaves GA that's doing any kind of tracking on the website.

I've updated the policy to match, so I think we're good to go. I'd like to merge this tomorrow so we can announce it in the newsletter - could I get a review? :)

ohadlevy reviewed (approved):
T H A N K   Y O U  @GregSutcliffe  great job!

ekohl commented on privacy.md:
```diff
@@ -0,0 +1,216 @@
+
+### Website videos
+
+The website does embed videos from YouTube, Fosdem, and Slideshare - see the
```
Should it be FOSDEM?

GregSutcliffe replied:
Fixed, ty

ekohl commented on privacy.md:
```diff
@@ -0,0 +1,216 @@
+
+### Website downloads
+
+The downloads from our website are cached by Fastly, a CDN provider. This may
```
s/cached/served/ perhaps? Might also be good to explicitly mention `downloads.theforeman.org` and `staging-deb.tf.o` since those are currently served by the CDN, not the main website.

GregSutcliffe replied:
I was trying to avoid having to update this policy once we expand the CDN coverage. I can change it if we think more specificity is needed.

ekohl replied:
I guess it's fine for now.

lzap commented on privacy.md:
```diff
@@ -0,0 +1,216 @@
+automatically), and the data is used by our developers to assist users in
+debugging complex problems. The data is not used for any analysis or
+statistics, is only accessible to a small number of the Foreman developers, and
+is deleted regularly.
```
Is this done automatically? I remember this was on our TODO list.

lzap reviewed (approved):
Nice writeup! IAANAL

tbrisker reviewed (approved):
I think we have enough +1s here and I didn't see any concerns that were not addressed, so i think we can merge and update in the future if needed.

tbrisker commented:
Thanks you very much @GregSutcliffe for all the hard work put into this!

GregSutcliffe commented:
Thanks all for the input. I'll get announcements out to the forum, newsletter, and community demo tomorrow.
